framework/include/base/FEProblemBase.h

```diff
@@ -665,6 +666,13 @@ class FEProblemBase :
   MooseSharedPointer<MultiApp> getMultiApp(const std::string & multi_app_name);
 
   /**
+   * Execute MultiAppTransfers associate with execution flag and direction.
+   * @param type The execution flag to execute.
+   * @param direction The direction (to or from) to transfer.
+   */
+  void execMultiAppTransfers(ExecFlagType type, MultiAppTransfer::DIRECTION direction);
```

Comment:
Yeah, I know you only added the name DIRECTION. I just figured since you touched that line of code you'd modernize it at the same time. Not strictly required but a good policy. Maybe we should formalize it.